docs/intentReaderHandlers/SetApiResponseHandler.md

```diff
@@ -0,0 +1,106 @@
+# SetApiResponseHandler 
+
+## Overview
+
+SetApiResponseHandler is invoked when the task specified in the intent has the value "setApiResponse". This handler is used to assign the apiResponse value to an environment variable in FCA.
+
+It performs the following actions:
```

Comment:
In my window this is not formatting the numbered list correctly:
1. Parses the input message received to get the API response. 2. Assigns the value to an environment variable.

Reply:
updated in required format